docs/changes/newsfragments/xxxx.breakling

```diff
@@ -0,0 +1,7 @@
+The `InstrumentChannel` class `AWGChannel` used by the `70001A', '70002A', '70001B', '70002B', '5208'
+tektronix awgs has changed name to `Tektronix70000AWGChannel` to conform with the naming of the instrument
+classes.
+
+The `InstrumentChannel` class ``TPS2012Channel`` has changed name to ``TektronixTPS2012Channel``
+
+The Tektronix DPO exception `ModeError` has been renamed `TektronixMSODPOModeError`
```

Comment:
let's preserve the `ModeError = TektronixMSODPOModeError` as alias? as is done for rigol?